common/envHandler.js

```diff
@@ -33,6 +33,39 @@ envHandler.get = function (envName, cb) {
   );
 };
 
+envHandler.post = function (envName, envValue, cb) {
+  if (envName === null || envValue === null)
```

Comment:
what if its undefined and not null? use udnerscore here

Reply:
@ric03uec updated to add checks using underscore.js. we won't reach this point from API because json parsing itself fails if payload contains `undefined` but added the checks anyway for safety.